installing/validating-an-installation.adoc

```diff
@@ -0,0 +1,61 @@
+
+//Reviewing the installation log
+include::modules/reviewing-the-installation-log.adoc[leveloffset=+1]
+
+//Getting cluster version and update details
+include::modules/getting-cluster-version-and-update-details.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../updating/updating-cluster-between-minor.adoc#updating-cluster-between-minor[Updating a cluster between minor versions] for more information on updating your cluster.
+
+* See xref:../updating/updating-cluster-between-minor.adoc#understanding-upgrade-channels_updating-cluster-between-minor[OpenShift Container Platform upgrade channels and releases] for an overview about upgrade release channels.
+
+//Querying the status of cluster components by using the CLI
+include::modules/querying-the-status-of-cluster-components-using-the-cli.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../support/troubleshooting/troubleshooting-operator-issues.adoc#troubleshooting-operator-issues[Troubleshooting Operator issues] for information about investigating issues with cluster Operators.
+
+* See xref:../support/troubleshooting/verifying-node-health.adoc#verifying-node-health[Verifying node health] for more details about reviewing node health and investigating node issues.
+
+* See xref:../support/troubleshooting/investigating-pod-issues.adoc#investigating-pod-issues[Investigating pod issues] for more details about reviewing pod health and investigating pod issues.
+
+//Reviewing the cluster status from the OpenShift Container Platform web console
+include::modules/reviewing-cluster-status-from-the-openshift-web-console.adoc[leveloffset=+1]
+
+//Reviewing the cluster status from the OpenShift Cluster Manager
+include::modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc#using-insights-to-identify-issues-with-your-cluster[Using Insights to identify issues with your cluster] for more information about reviewing potential issues with your cluster.
+
+//Checking cluster resource availability and utilisation
+include::modules/checking-cluster-resource-availability-and-utilisation.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack[Understanding the monitoring stack] for more information about the {product-title} monitoring stack.
+
+//Listing alerts that are firing
+include::modules/listing-alerts-that-are-firing.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../monitoring/managing-alerts.adoc#managing-alerts[Managing alerts] for further details about alerting in {product-title}.
+
+== Next steps
+
+* xref:../support/troubleshooting/troubleshooting-installations.adoc#troubleshooting-installations[Troubleshooting installations]
```

Comment:
Is this actually a required next step?

Or maybe preface with _Optional:_

Reply:
That is a good point. I have subsequently updated that line as follows:

* See xref:../support/troubleshooting/troubleshooting-installations.adoc#troubleshooting-installations[Troubleshooting installations] if you experience issues when installing your cluster.